### ISSUE TITLE
windows pypy/4.2.0

### ISSUE THREAD

**Issue by wiredfool** [python-pillow/Pillow#2607]
Hoisted from https://github.com/python-pillow/Pillow/issues/2567#issuecomment-312707616

@CTPaHHuK-HEbA commented:

Windows, pypy:
```
  File "C:\Python27\pypy\site-packages\PIL\Image.py", line 63, in <module>
    PILLOW_VERSION))
ImportError: The _imaging extension was built for another version of Pillow or PIL: Core Version: "4.2.0"Pillow Version:  4.2.0
```
'"4.2.0"' != '4.2.0'

----

Apparently the distutils on pypy are different on windows, and we're not testing pypy on appveyor. 

@cgohlke mentioned that the pypy build was non-functional, perhaps this is the reason.

**Comment by cgohlke** [comment]
> @cgohlke mentioned that the pypy build was non-functional, perhaps this is the reason.

Yes, this is the same error I got.

**Comment by cgohlke** [comment]
> This should be enough to fix it: wiredfool/Pillow@3f3eb99

Works for me. Thanks.
